pilot/pkg/security/authz/matcher/string_test.go

```diff
@@ -70,3 +69,48 @@ func TestStringMatcherWithPrefix(t *testing.T) {
 		})
 	}
 }
+
+func TestStringMatcherRegex(t *testing.T) {
+	testCases := []struct {
+		name string
+		v    string
+		want *matcherpb.StringMatcher
+	}{
+		{
```

Comment:
This looks a little bit duplicate, can you follow the other test cases to define a `testCases` for all test cases, see the example in line 25.

Reply:
I de-dup the two func with a global <code>testCase</code> struct. 
Is this a good way to avoid the verbose declaration?

Reply:
yes, this is fine as the testCase is not exported.